src/test/java/org/broadinstitute/consent/http/resources/DarCollectionResourceTest.java

```diff
@@ -47,6 +40,9 @@
     new UserRole(UserRoles.RESEARCHER.getRoleId(), UserRoles.RESEARCHER.getRoleName())
   );
   private final User researcher = new User(1, authUser.getEmail(), "Display Name", new Date(), researcherRole, authUser.getEmail());
+  private final List<UserRole> signingOfficialRole = Collections.singletonList(
```

Comment:
`List.of` is a little easier to read. Was added in later versions of java.

```suggestion
  private final List<UserRole> signingOfficialRole = List.of(
```